kratos/utilities/mortar_utilities.h

```diff
@@ -181,8 +181,8 @@ class MortarUtilities
         array_1d<double, 3> current_global_coords;
 
         array_1d<array_1d<double, 3>, 2> normals;
-        normals[0] = GeomOrigin[0].GetValue(NORMAL);
-        normals[1] = GeomOrigin[1].GetValue(NORMAL);
+        normals[0] = GeomOrigin[0].FastGetSolutionStepValue(NORMAL);
```

Comment:
this is also in the core. You can not change it as you wish...

Reply:
Right now I am the only one using it. I put it in here for the future developments of the MappingApplication, not yet used

Reply:
same comment. Then be sure that the tests are there. i will accept for this time, but next time you want to change it should be in a separate PR